base/operators.jl

```diff
@@ -1149,40 +1149,68 @@ julia> filter(!isletter, str)
 !(f::ComposedFunction{typeof(!)}) = f.inner #allows !!f === f
 
 """
-    Fix1(f, x)
+    Fix{n}(f, x)
+    Fix{:kw}(f, x)
+    Fix(f; kw=x)
 
-A type representing a partially-applied version of the two-argument function
-`f`, with the first argument fixed to the value "x". In other words,
-`Fix1(f, x)` behaves similarly to `y->f(x, y)`.
+A type representing a partially-applied version of a function `f`, with the argument
+"x" fixed at argument `n::Int` or keyword `kw::Symbol`.
+In other words, `Fix{3}(f, x)` behaves similarly to
+`(y1, y2, y3) -> f(y1, y2, x, y3)` for the 4-argument function `f`.
```

Comment:
Hm yeah this might be confusing. What I meant to say was: if you have a 4-argument function `f`, then `Fix{3}(f, x)` is equivalent to `(y1, y2, y3) -> f(y1, y2, x, y3)`. But of course if it had a different number of arguments, it would be different.

Not sure the best phrasing here; tips appreciated.

Reply:
```suggestion
`(y1, y2, y3...) -> f(y1, y2, x, y3...)`.
```